src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderMovementsResource.java

```diff
@@ -81,6 +81,19 @@ OffenderBooking moveToCell(
             @ApiParam(value = "The date / time of the move (defaults to current)", example = "2020-03-24T12:13:40") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) @RequestParam(value = "dateTime", required = false) LocalDateTime dateTime
     );
 
+
+    @PutMapping("/{bookingId}/move-to-cell-swap")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = OffenderBooking.class),
+            @ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class),
+            @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
+    OffenderBooking moveToCellSwap(
+            @ApiParam(value = "The offender booking id", example = "1200866", required = true) @PathVariable("bookingId") Long bookingId,
+            @ApiParam(value = "The reason code for the move (from reason code domain CHG_HOUS_RSN)", example = "ADM", required = true) @RequestParam("reasonCode") String reasonCode,
```

Comment:
Would a RequestBody be more appropriate for tis PUT rather than parameters?

Reply:
Now you mention it, yeah it would definitely feel more natural.

Reply:
I'll address it in the next PR as I know we'll need to make some changes once we've played with it a bit.